### ISSUE TITLE
Exception when trying to open a virtual drive (Python 3.5.3 issue on Windows 10)

### ISSUE THREAD

**Issue by sytsereitsma** [fman-community/fman#320]
I use BoxCryptor to encrypt some of my DropBox folders. BoxCryptor creates a virtual drive on my system in which it presents the unencrypted data. When I try to open the drive, or any folder on the drive I get an exception (workaround below):

```python
Command 'GoTo' raised error.

Traceback (most recent call last):
  File "fman\impl\plugins\plugin.py", line 289, in _run_in_thread
  File "C:\Users\sytse\AppData\Local\fman\Versions\0.8.9\Plugins\Core\core\commands.py", line 827, in __call__
    self.pane.set_path(url)
  File "fman\__init__.py", line 107, in set_path
  File "fman\impl\util\qt\thread.py", line 11, in result
  File "fman\impl\util\qt\thread.py", line 58, in run_in_thread
  File "fman\impl\util\qt\thread.py", line 83, in result
  File "fman\impl\util\qt\thread.py", line 71, in __call__
  File "fman\impl\widgets.py", line 107, in set_location
  File "fman\impl\model\__init__.py", line 550, in set_location
  File "fman\impl\model\__init__.py", line 172, in set_location
  File "fman\impl\plugins\mother_fs.py", line 105, in resolve
  File "C:\Users\sytse\AppData\Local\fman\Versions\0.8.9\Plugins\Core\core\fs\local\__init__.py", line 81, in resolve
    url = as_url(Path(path).resolve())
  File "pathlib.py", line 1109, in resolve
  File "pathlib.py", line 186, in resolve
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'X:\\'
```
It appears this issue pertains to Python 3.5.3 only (I could not reproduce the error on Python 3.6).
I modified ```fman\Versions\0.8.9\Plugins\Core\core\fs\local\__init__.py``` to work around this issue by catching the error and checking if indeed the file does not exist:

```python
	def resolve(self, path):
		if not path:
			raise filenotfounderror(path)
		# Unlike other functions, Path#resolve can't handle C: instead of C:\
		path = self._add_backslash_to_drive_if_missing(path)
		try:
			url = as_url(Path(path).resolve())
		except FileNotFoundError:
			#This can happen for virtual drives on Windows 10 with Python 3.5.3
			if PLATFORM == 'Windows' and os.path.exists(path):
				url = as_url(path)
			else:
				raise
		return url
```

Sytse

**Comment by mherrmann** [comment]
Interesting, thank you for the research and for sharing the fix.

By "the issue pertains to Python 3.5.3 only", you probably mean that you are able to reproduce it in a Python 3.5.3 shell, but not in a 3.6 one? Because the Python version which fman uses is fixed (3.5 atm).

**Comment by sytsereitsma** [comment]
You are correct. I tried Path.resolve in both shells. 3.5.3 has the issue, 3.6 does not.

**Comment by mherrmann** [comment]
Okay! Thanks :)

**Comment by mherrmann** [comment]
I just released [fman 0.9.1](https://fman.io/changelog#v0.9.1) that updates Python to 3.6. It should fix the issue :-) (@sytsereitsma, if it doesn't, could you let me know?)

**Comment by mherrmann** [comment]
Re-opening this issue because fman's Python version was [downgraded](https://fman.io/changelog#v0.9.2) to Python 3.5.

**Comment by mherrmann** [comment]
Fixed in [fman 1.0.9](https://fman.io/changelog#v1.0.9).
